cmd/minishift/cmd/start_preflight.go

```diff
@@ -58,6 +58,12 @@ func preflightChecksBeforeStartingHost() {
 			"Checking if KVM driver is installed",
 			false, configCmd.WarnCheckXHyveDriver.Name,
 			driverErrorMessage)
+		preflightCheckSucceedsOrFails(
+			configCmd.SkipCheckKVMDriver.Name,
+			checkDefaultNetwork,
+			"Checking if default network is present",
+			false, configCmd.WarnCheckXHyveDriver.Name,
```

Comment:
Xhyve ?

Reply:
which default network.... can this be more specific?
I see people think... `default` network? I know of default gateway?!

Reply:
i saw the other check also used  `configCmd.WarnCheckXHyveDriver.Name` so..